src/tr1/game/objects/creatures/bear.c

```diff
@@ -76,6 +71,8 @@ static void M_Setup(OBJECT *const obj)
 
 static void M_Control(const int16_t item_num)
 {
+    OBJECT *const obj = Object_Get(O_BEAR);
```

Comment:
One last nitpick - can we use `item->object_id`?

Reply:
Of course, my bad. I was a bit hasty with this 😄